### DISCUSSION TITLE
Some Suggestions!

### DISCUSSION THREAD

**Discussion by Slendykins** [BudgetAirpods/InsanityRemastered#19] [Ideas]
Love what this mod is achieving! I like the more spooky side of LC and with a lot of playtime i'd like to give some thought for what could possibly be added to the mod!

I've got a couple suggestions which may help spark some ideas;

- An insanity drain scale value, one at the top level that will reduce or increase the rate at which everything affects sanity
- Double down on Skinwalker mod integration, the "player" hallucination could also play these voicelines?
- "Player" hallucination suit colour, like what MaskedEnemyOverhaul does, changes the "Masked" to use player equipped suit colours
- Panic attack jumpscare? Have a "player" or "bracken" hallucination in the distance, upon looking away they'll appear behind you for a second or 2, when turning around in that timeframe it'll freak the player out
- I got the "player" hallucination but it was pretty obvious it wasn't someone due to it walking towards me, maybe something freaky like a run then masked grab fakeout?

I'll think of some more as we get a good bit more experience with the mod, but so far it's been great!

**Comment by BudgetAirpods** [comment]
> Love what this mod is achieving! I like the more spooky side of LC and with a lot of playtime i'd like to give some thought for what could possibly be added to the mod!
> 
> I've got a couple suggestions which may help spark some ideas;
> 
> * An insanity drain scale value, one at the top level that will reduce or increase the rate at which everything affects sanity
> * Double down on Skinwalker mod integration, the "player" hallucination could also play these voicelines?
> * "Player" hallucination suit colour, like what MaskedEnemyOverhaul does, changes the "Masked" to use player equipped suit colours
> * Panic attack jumpscare? Have a "player" or "bracken" hallucination in the distance, upon looking away they'll appear behind you for a second or 2, when turning around in that timeframe it'll freak the player out
> * I got the "player" hallucination but it was pretty obvious it wasn't someone due to it walking towards me, maybe something freaky like a run then masked grab fakeout?
> 
> I'll think of some more as we get a good bit more experience with the mod, but so far it's been great!

1. I'll consider that.
2. I'm just now realizing I forgot to add this in with the recent update which adds skinwalker integration. Oopsies.
3. I actually had this in mind when I started working on the player hallucination, but I ended up scrapping it for some reason (it might be because I just didn't know how to go about doing it.) This is something that I'll have on my top priority, and I will have it work with mods like MoreSuits.
4. This is an interesting idea. I'll think about adding this in.
5. Yeah, I'm aware how obvious it is when it's a fake player hallucination. I'm thinking of reworking the AI to be a little more believable.

I really love the horror side of LC as well. I made this mod especially because I love horror games and wanted to expand on the already spooky atmosphere of LC. Thank you for your suggestions.

**Comment by Slendykins** [comment]
If it's any help the modding Discord is full of approachable individuals, HomelessGinger who made MaskedEnemyOverhaul will surely help with any quirks around referencing player suits!

**Comment by BudgetAirpods** [comment]
This has been added! It was really simpler than I thought it would be.

**Comment by Aleph-Zero0** [comment]
You could also add nametags to the fake players
